src/transforms/select.jl

```diff
@@ -2,29 +2,43 @@
 # Licensed under the MIT License. See LICENSE in the project root.
 # ------------------------------------------------------------------
 
+const ColSelector = Union{Vector{Symbol}, Regex}
```

Comment:
What about renaming this to `ColSpec`? It solves the Select/Reject nomenclature issue.

Reply:
I think that's a good name.
I will rename `ColSelector` to `ColSpec`.

Reply:
Done!